README.md

```diff
@@ -1,7 +1,33 @@
-# SKlauncher
+<img src="https://skmedix.pl/images/logo.png" alt="SKlauncher logo" align="right" width="256" height="256"/>
 
-Useful links
----
-* [Official SKlauncher website](https://skmedix.pl)
-* [Discord server](https://discord.gg/BdCcpDZ)
-* [Language codes (ISO 639-3)](https://minecraft.wiki/w/Language#Languages)
+# SKlauncher #
+
+[![SKlauncher Website][site-shield]][site-url]
+[![Discord Server][discord-shield]][discord-url]
+[![GitHub Forks][github-shield]][github-url]
+
+[SKlauncher][site-url] is a game launcher for
+the game Minecraft: Java Edition.
+
+This repo hosts translation files for SKlauncher.
+
+## Contributing ##
+
+Thank you for your interest in translating SKlauncher!
+
+Please check out the [Contributions Guide][guide-url],
```

Comment:
I've used the new documentation from the
official website <https://docs.skmedix.pl>.

I think the wiki in this repo is out of
date, maybe it should redirect there.